### PR TITLE
wip: realm_test fix/improvement attempt

### PR DESCRIPTION
**Pull request by agagancarczyk**
## Motivation
An attempt at fixing/improving the "should create Test Disabled realm" test detected by Cypress Dashboard as flaky.

realm_test suit is passing locally:

Screenshot 2022-04-12 at 15 57 27

## Verification Steps
1. Run tests in `realm_test.spec.ts` locally.

## Checklist:

- [ ] Code has been tested locally by PR requester
- [ ] User-visible strings are using the react-i18next framework (useTranslation)
- [ ] Help has been implemented
- [ ] axe report has been run and resulting a11y issues have been resolved
- [ ] Unit tests have been created/updated

### CODE REVIEW

File: cypress/support/pages/admin_console/CreateRealmPage.ts [hunk 1]
```diff
@@ -1,8 +1,6 @@
 export default class CreateRealmPage {
-  private browseBtn = "#kc-realm-filename-browse-button";
```
Comment (jonkoops) [inline cypress/support/pages/admin_console/CreateRealmPage.ts:2]: Nice cleanup!

**Review by jonkoops — CHANGES_REQUESTED** [review]
Changes look good, but it looks like the test still fails.

File: cypress/integration/realm_test.spec.ts [hunk 2]
```diff
@@ -66,6 +66,7 @@ describe("Realms test", () => {
 
     it("should create Test Disabled realm", () => {
       sidebarPage.goToCreateRealm();
+      sidebarPage.waitForPageLoad();
```
Comment (edewit) [inline cypress/integration/realm_test.spec.ts:69]: `goToCreateRealm` already does a `waitForPageLoad`

Comment (agagancarczyk) [reply cypress/integration/realm_test.spec.ts:69]: thanks @edewit it seems to be needed. Would you recommend using something else here in the form of waiting?

**Review by jonkoops — APPROVED** [review]
Changes look good to me.

File: cypress/integration/realm_test.spec.ts (hunk 2 above)
Comment (edewit) [reply cypress/integration/realm_test.spec.ts:69]: ahh okay I see... there are 2 loading spinners... no, it's fine like this

**Pull request merged**